packages/micropip/src/micropip/_micropip.py

```diff
@@ -410,6 +419,11 @@ def install(requirements: str | list[str], keep_going: bool = False):
         - If ``True``, the micropip will keep going after the first error, and report a list
           of errors at the end.
 
+    deps : ``bool``, default: True
+
+        If ``True``, install dependencies of the given packages specified in METADATA file.
+        If ``False``, install the given packages without dependencies.
```

Comment:
```suggestion
        If ``True``, install dependencies specified in METADATA file for
        each package. Otherwise do not install dependencies.
```